docs/esterni/analisi_dei_requisiti/analisi_dei_requisiti.tex

```diff
@@ -286,8 +293,8 @@ \subsection{Attori}
 \textbf{Estensione}:
 UC8.1 Visualizzazione messaggio procedura di pagamento annullata:
 \begin{enumerate}
-    \item visualizzazione messaggio procedura di pagamento annullata;
     \item cancellazione sessione di pagamento.
+    \item l'acquirente viene indirizzato presso l'home page di \textit{google.com}.
```

Comment:
No. L'acquirente viene reindirizzato alla pagina dell'ecommerce da cui ha appena pagato, e si passa alla suddetta pagina in GET l'id dell'avvenuta transazione in modo tale che l'ecommerce possa salvarselo.